cmd/plugin/commands/ssh/ssh.go

```diff
@@ -50,6 +52,11 @@ func ssh(flags *genericclioptions.ConfigFlags, podName string, deployment string
 	if err != nil {
 		return err
 	}
-
-	return kubectl.Exec(flags, []string{"exec", "-it", "-n", pod.Namespace, "-c", container, pod.Name, "--", "/bin/bash"})
+	fmt.Printf("exec in %s\n", pod.Name)
```

Comment:
If you want to add log here, I think it would be better to log namespace together

Reply:
well, this is done.